code/datums/spells/mind_transfer.dm

```diff
@@ -43,6 +43,10 @@ Also, you never added distance checking after target is selected. I've went ahea
 		user << "<span class='warning'>They appear to be catatonic! Not even magic can affect their vacant mind.</span>"
 		return
 
+	if(user.suiciding)
+		user << "<span class='warning'>You're killing yourself! You can't concentrate enough to do this!</span>"

```

Comment:
"the wizard federation is most displeased with your dishonorable acts and has formally decided to smite you"

dust()